### PR TITLE
Blog app -  Controllers, Routes, and  Views

### PR DESCRIPTION
**Pull request by kit0-0**
## Description
This pull request addresses the initial setup of controllers, routes, and placeholder views for the users and posts sections, following the project requirements. The changes include:

- Created controllers for Users and Posts with index and show actions.
- Added  HTML content for the index and show views of Users and Posts.
- Established routes to handle the specified URLs.
- Ensure the routes are correctly configured using `rails routes`.

image

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @kit0-0, please update your pull request title to be related to this current milestone, for this milestone you are creating controllers, views and routes 👍🏿

**Comment by prolajumokeoni** [comment]
Hi @kit0-0 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a descriptive pull request title 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get '/', to: 'users#index', as: :users
+  get '/users/:id', to: 'users#show', as: :user
+  get '/users/:id/posts', to: 'posts#index', as: :user_posts
+  get '/users/:user_id/posts/:id', to: 'posts#show', as: :user_post
```
Comment (Olamarx) [inline config/routes.rb:5]: - [ ] [OPTIONAL] I will encourage you to make use of resources for less code, you can achieve that with just three lines of code:

```
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

**Review by Olamarx — APPROVED** [review]
Hi @kit0-0,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- Routes setup ✔️
- Controllers methods ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**